templates.go

```diff
@@ -10,8 +10,16 @@ import (
 	"strings"
 )
 
+var funcMap = template.FuncMap{
+	"escape": func(text string) template.HTML {
```

Comment:
might be good to rename this to something more specific - `escape` seems quite generic (where as this is specific to escaping-for-windows)

Reply:
It's exactly as you say. I've renamed this key.